src/library/export/trackexportworker.h

```diff
@@ -49,6 +49,12 @@ class TrackExportWorker : public QThread {
         return m_errorMessage;
     }
 
+    //Get the list
+    QList<QString> getMissingTracks();
+
+    //Boolean that lets the worker know if the job will be skipped due to missing files
```

Comment:
If you need a long comment to explain the naming of a variable the naming is inappropriate. How about `m_canceledByUser`? It doesn't matter why.

Reply:
much better thanks!